src/ConstantSum/ConstantSum.sol

```diff
@@ -41,6 +41,7 @@ contract ConstantSum is IStrategy {
     function init(
         address,
         uint256 poolId,
+        IDFMM.Pool calldata pool,
```

Comment:
What is this for?

Reply:
yeah, it's not entirely clear to me why we're passing the pool in `init`, I checked the implementations in all of the strategies and none of them seem to use this argument.

Reply:
I explained this on Discord or during a standup, there are two main reasons:
1. Most of the other functions in `IStrategy` are receiving the `pool` parameter, so I added it for the sake of conformity
2. This extends the possibilities of the strategies, for example restricting the pool creation to stable tokens only, etc...